plugins/modules/fusion_pg.py

```diff
@@ -276,21 +277,26 @@ def main():
     state = module.params["state"]
     pgroup = get_pg(module, fusion)
 
+    id = None
+    if pgroup is not None:
+        id = pgroup.id
+
     if state == "present" and not pgroup:
         module.fail_on_missing_params(
             ["region", "availability_zone", "storage_service"]
         )
-        changed = create_pg(module, fusion) or changed
+        changed, id = create_pg(module, fusion) or changed
         if module.params["array"]:
             # changing placement requires additional update
             pgroup = get_pg(module, fusion)
-            changed = update_pg(module, fusion, pgroup) or changed
+            changedUpdate = update_pg(module, fusion, pgroup)
+            changed = changed | changedUpdate
     elif state == "present" and pgroup:
         changed = update_pg(module, fusion, pgroup) or changed
     elif state == "absent" and pgroup:
         changed = delete_pg(module, fusion) or changed
 
-    module.exit_json(changed=changed)
+    module.exit_json(changed=changed, id=id)
```

Comment:
Why in some modules you have if statement for id != None and here you don't do check. Maybe also add if id != None?